ddp-workspace/projects/ddp-testboston/src/assets/i18n/en.json

```diff
@@ -268,7 +272,7 @@
     }
   },
   "Enrollment": {
-    "Title": "Account setup",
+    "Title": "Account set-up",
```

Comment:
The hyphen in `set-up` is unusual in American English as far I know. Original looks more correct to me.

Reply:
I changed that per Jen's request:
![image](https://user-images.githubusercontent.com/30126907/89262212-021a1d80-d638-11ea-82f3-57f5ff517208.png)
![image](https://user-images.githubusercontent.com/30126907/89262285-0fcfa300-d638-11ea-9558-4547dde56f9f.png)